src/Config/Config.php

```diff
@@ -231,6 +231,9 @@ class Config
 
     const CARTES_BANCAIRES = 'cartesbancaires';
 
+    const MODULE_UPGRADE_NOTICE_CLOSE_DATE = 'MODULE_UPGRADE_NOTICE_CLOSE_DATE';
+    const MODULE_UPGRADE_NOTICE_CLOSE_DURATION = '28 days';
```

Comment:
Const name is okay, but configs name should be something unique to your module. For example MOLLIE_MODULE_UPGRADE_NOTICE_CLOSE_DATE 

There is always a chance that some other module decides to name their config 'MODULE_UPGRADE_NOTICE_CLOSE_DATE'